modules/data_integrity_flag/jsx/setFlagForm.js

```diff
@@ -68,22 +68,24 @@ class SetFlagForm extends Component {
   }
 
   render() {
+    const {fieldOptions} = this.props;
```

Comment:
what's the reason for aliasing this.props with fieldOptions?

Reply:
This is actually assigning each prop to an individual variable. It can be extended to every single prop. E.g.
```
const {prop1, prop2, prop3} = this.props;
```
This is pretty useful in drastically uncluttering the code.

Reply:
destructuring, not destruction.

Reply:
😮

Reply:
but why wouldn't you just use for example this.props.visits down below on line 80 ?

Reply:
It would have to be `this.props.fieldOptions.visits`.

Reply:
ahh i see, so its for cleaner code. gotcha!